src/js/components/Pagination/StyledPageControl.js

```diff
@@ -3,54 +3,17 @@ import styled from 'styled-components';
 import { Button } from '../Button';
 import { Text } from '../Text';
 
-const BUTTON_STATES = ['normal', 'active', 'disabled', 'hover'];
-
-// Assemble button theme object for each button state
-const buildButtonStyle = (buttonTheme, kind) => {
-  const style = buttonTheme[kind];
-  BUTTON_STATES.forEach(state => {
-    if (state !== 'normal' && buttonTheme[state])
-      style[state] = buttonTheme[state][kind];
-  });
-  return style;
-};
-
```

Comment:
😮  This makes a HUGE difference!

Reply:
Yes ! It was very nice to hit delete on all of that : )